docs/testing-deployment-publishing/exporting-code/ff-cli.md

```diff
@@ -0,0 +1,69 @@
+---
+slug: ff-cli
+title: FlutterFlow CLI
+tags: []
+description: Learn how to download and manage your FlutterFlow projects locally using the FlutterFlow CLI.
+sidebar_position: 1
+---
+
+# FlutterFlow CLI
+
+The [FlutterFlow CLI](https://pub.dev/packages/flutterflow_cli) tool allows you to quickly and easily download your project files directly from FlutterFlow to your local machine.
+
+## Installation
+
+To use the FlutterFlow CLI, you first need to install it globally using Dart's package manager with the following command:
+
+```json
+dart pub global activate flutterflow_cli
+```
+
+## Usage
+
+Follow the steps below to export your project.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'
+}}>
+    <iframe 
+        src="https://demo.arcade.software/Rc3s1P8DFypUKoPzVITL?embed&show_copy_link=true"
+        title="Sharing a Project with a User"
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+
+### CLI command details
+
+- If you wish to exclude assets from the download, use `-no-include-assets` in your command. This will download the project code without the assets. For example: `flutterflow export-code --project your_project_id --dest path_to_output_folde --no-include-assets --token your_token`
+    
+- You can download code from a specific branch by switching to that branch and using the toolbar command, or by including the `-branch-name` or `-b` flag in your command and specifying the branch you wish to download from.
+
+#### All supported command options
+
+| Flag | Behavior | Default |
+| --- | --- | --- |
+| --dest / -d | Specifies a destination folder other than the current directory | Current directory |
+| --[no]-include-assets | Option to download assets (images, GIFs). Useful for consecutive code exports if the assets folder hasn't changed | False |
+| --branch-name / -b | Downloads from a specific branch | Main |
+| --[no]-fix | Option to run dart fix on the code after downloading | False |
+| --[no]-parent-folder | Option to download the code into a subfolder instead of directly into the directory | False |
+
+## Filtered exports
```

Comment:
This can also be added as a reference (under Tips) in Github Manage Custom Code section. Like you can reference users to this section and page so they can read more about CLI tool. This tip can be end of Manage Custom Code section

Reply:
Done!